### PR TITLE
[MRG] Shorthand for prior of Real variables

### PR DESCRIPTION
**Pull request by glouppe**
This PR adds the shorthand `(lower, upper, prior)` for defining `Real` parameters. Also includes some cosmit changes.

### CODE REVIEW

File: skopt/parameter.py [hunk 1]
```diff
@@ -100,8 +80,29 @@ def inverse_transform(self, values):
         return self._lb.inverse_transform(values)
 
 
+class Distribution:
+    def rvs(self, n_samples=None, random_state=None):
+        """
+        Randomly sample points from the original space
+        """
+        random_vals = self._rvs.rvs(size=n_samples, random_state=random_state)
+        return self.transformer.inverse_transform(random_vals)
```
Comment (MechCoder) [inline skopt/parameter.py]: I don't know why but does `self.inverse_transform(random_vals)` look better?

File: skopt/parameter.py [hunk 2]
```diff
@@ -100,8 +80,29 @@ def inverse_transform(self, values):
         return self._lb.inverse_transform(values)
 
 
+class Distribution:
+    def rvs(self, n_samples=None, random_state=None):
+        """
+        Randomly sample points from the original space
```
Comment (MechCoder) [inline skopt/parameter.py]: Should we document `n_samples` and `random_state` here?

**Comment by MechCoder** [comment]
Just some minor comments. Otherwise LGTM

**Comment by glouppe** [comment]
Thanks! Comments addressed. Merging.

**Pull request merged**